src/features/search/components/SearchBox.tsx

```diff
@@ -0,0 +1,47 @@
+import React, { useState } from 'react'
+import { TouchableOpacity } from 'react-native-gesture-handler'
+import styled from 'styled-components/native'
+
+import { SearchInput } from 'ui/components/inputs/SearchInput'
+import { Invalidate } from 'ui/svg/icons/Invalidate'
+import { MagnifyingGlass } from 'ui/svg/icons/MagnifyingGlass'
+
+interface Props {
+  onChangeText?: (text: string) => void
```

Comment:
Si c'est plus simple pour toi, ne pas hésiter :
```suggestion
  onChangeText: (text: string) => void
```

Reply:
quand on wrappera SearchBox avec le connecteur algolia, il sera plus optionnel, mais pour l'instant c'est pour éviter de passer un fonction vide.